static/app/components/searchQueryBuilder/valueCombobox.tsx

```diff
@@ -613,15 +774,13 @@ export function SearchQueryBuilderValueCombobox({
     [canSelectMultipleValues, dispatch, token]
   );
 
-  // Clicking anywhere in the value editing area should focus the input
-  const onClick: React.MouseEventHandler<HTMLDivElement> = useCallback(e => {
-    if (e.target === e.currentTarget) {
-      e.preventDefault();
-      e.stopPropagation();
-      inputRef.current?.click();
-      inputRef.current?.focus();
-    }
-  }, []);
+  const onKeyUp = useCallback(() => {
+    updateSelectionIndex();
+  }, [updateSelectionIndex]);
+
+  const onClick = useCallback(() => {
+    updateSelectionIndex();
+  }, [updateSelectionIndex]);
```

Comment:
just reuse the same funciton?

Reply:
Not sure why I didn't do that to begin with